addons/l10n_at/data/account_tax_template.xml

```diff
@@ -1672,14 +2022,13 @@
             12 Abs. 1 Z 2 lit. a) -->
         <record id="account_tax_template_purchase_xx_code061" model="account.tax.template">
             <field name="name">VST_061 entrichtete EUst (§ 12 Abs. 1 Z 2 lit. a)</field>
-            <field name="description">VSt. 20%</field>
+            <field name="description">20%</field>
             <field name="chart_template_id" ref="l10n_at_chart_template" />
             <field name="sequence">400</field>
             <field name="type_tax_use">purchase</field>
-            <field eval="20" name="amount" />
+            <field eval="100" name="amount" />
```

Comment:
how?

Reply:
Before, it was not properly defined at all and here we have the use case that the carrier is taking over your duty to pay the tax to the ministry of finance in the name of the company, therefore it is forwarded (purely the tax). 
As we are allowed to deduct the whole amount in terms of deductible VAT, it is like that an properly encoded on the related accounts.

Same is true for 083 where we do get an information by the ministry of finance that we have to pay the "sale tax". Based on this we do an extra tax account move to be able to deduct the "purchase tax" again.

One is directly on an incoming invoice (061) and the other one (083) needs to be done separately in an own account move.

Hope this helps, but 100% correct and used in several companies and checked with tax lawyers 😉

Reply:
Where do you need to report the base amount?  (normally doing a group of taxes for Spain) 
Because if no need for reporting, we could use a division type.

Reply:
Puh, difficult to relate to Spain as I am not used to what they have or have not to do and also I have not a lot of experience with Spanish companies but I can remember we discussed this before and I was convinced that the way I do it is, but feel free to give me a call and we will try to elaborate together once again. You might have a point, but as this is a "complicated" tax use case, I can not just say, yes, you are right, I have totally to rework this now after using it for a long time in production and redefining also several times to simplify for the usage and daily routine of the accountant user in Austria 😉

Reply:
You are not talking about the base amount, so I suppose we can just put this tax as a division included one, so there is no need for an extra negative line and the line becomes just tax.

Reply:
@jco-odoo 
IMO, I need the taxes in different `account.move.line` than the line with the 100% base amount.

![image](https://github.com/odoo/odoo/assets/362478/6f46919e-f8e5-48c6-a628-7357167a0e53)

Here is what I expect from 083 and 061 whereas the latter can be created from an purchase invoice line.

Would your suggestion create the exact same `account.move`?

Reply:
![image](https://github.com/odoo/odoo/assets/3832699/2f2d8f62-31d1-4217-93bf-f7806ffcdfbd)

Reply:
Not the same IMO and having a `account.move.line` with 0 debit and 0 credit is also awkward.

Reply:
Maybe it helps that in `083` the deductible VAT must be claimed via the report whereas the applied tax is claimed on customs import directly...so, in the end `3530` will be reconciled / reduced with the amount being paid or getting back from the tax authority.
As said, it is "complicated", not sure how we can settle this here on Github w/o you believing me that I need it as it is 😄
Still open to try to explain it to you in person again, but maybe just accepting my solution would make me happier I guess.

Reply:
The product line does represent 100% of the tax as you will never find a base amount in terms of purchase. This tax is mostly used on carrier invoices who are forwarding the tax for us to the tax authority which is based on the value of the goods imported and bill us for exactly this amount to get their money back. The value of the goods will not be part of the invoice lines.

The 20% is the VAT applied on the value of the goods (including insurance, transport costs and product costs), so our 100% will represent 20% of the "invisible" amount (not represented on the invoice in accounting), maybe as a side note somewhere in the carrier invoice depending on the carriers IT.

I hope this helps to understand.